### DISCUSSION TITLE
Mirth Maps Available to Postprocessor Script?

### DISCUSSION THREAD

**Discussion by flugruger** [nextgenhealthcare/connect#5062] [Q&A]
Can you tell me which Mirth maps are available to the postprocessor script?

**Comment by TexasGaidheal** [comment]
According to the 3.12 user guide, page 390, "The following variable maps exist throughout Mirth Connect":
Response Map, Connector Map, Channel Map, Source Map, Global Channel Map, Global Map, and Configuration Map.

This isn't the clearest wording, but it seems to say that all of these should work in the postprocessor script. (It would help if the user guide had a detailed index.)

**Comment by mlarcelet** [comment]
As far as I know, you cannot access channelMap or connectorMap from a post processing script. 
All other (configuration, globalChannel, global and response) can be used.
